docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/global/Common.scss

```diff
@@ -91,6 +91,7 @@
 @import '~Core/styles/alternating-lists';
 @import '~Core/styles/special-report-raw';
 @import '~Core/styles/title-first-feature-cards';
+@import '~Core/styles/sbir';
```

Comment:
Can you remove the reference to SBIR in commit 1f927bdd27c8de99f539c9f523bb4670859c77d1. This is not part of ticket 3311.